From 53bc3311a6f7f0d742a3500633be2c4209b462ae Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 5 Jun 2018 11:50:07 -0700
Subject: [PATCH] MANTA-3684 manta-scripts needs to work around default
 logadm.conf in new base image Reviewed by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com> Approved by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com>

---
 deps/manta-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/manta-scripts b/deps/manta-scripts
index dc593cd..cd8d6f1 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit dc593cd90475e391c6fce874130f5d522dc0cc54
+Subproject commit cd8d6f194ba34dfaa4b0740dfbd11147041dd13e
-- 
2.21.0

